src/Microsoft.Extensions.SecretManager.Tools/Internal/DotNetCoreSdk.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.ProjectModel.Internal
+{
+    internal class DotNetCoreSdk
```

Comment:
as far as i remember, we keep the types in `*.Internal` public. it is to avoid over-internalized types but explicitly states that types in `*.Internal` namespace are subject to change.